### PR TITLE
add pretest script to log versions

### PR DESCRIPTION
**Pull request by jdelbick**
## Description

Over the past few weeks, dep mod tests have been failing for various reasons:
- tools needed for the sdk not installed
- aio v8 being used

Adding this script to log the versions will help debug issues like this in the future.

related to ticket: https://jira.corp.adobe.com/browse/NUI-1502

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: check-versions.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2020 Adobe. All rights reserved.
```
Comment (tmathern) [inline check-versions.sh:3]:
```suggestion
# Copyright 2021 Adobe. All rights reserved.
```

File: check-versions.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# exit when any command fails
+set -e
+
+# log aio version
+aio info
+
+# log imagemagick version
+echo "....................................................................................................."
+identify --version
+echo "....................................................................................................."
```
Comment (tmathern) [inline check-versions.sh:22]: Add also other tools we use, like `exiftool`, so we have everything.

**Review by tmathern — CHANGES_REQUESTED** [review]

File: check-versions.sh (hunk 2 above)
Comment (jdelbick) [reply check-versions.sh:22]: I couldn't figure out how to get the exiftool version so the best I did was run exiftool on a file to check its installed: https://github.com/adobe/asset-compute-sdk/blob/019e4e48262fd04f30f0d9e2ff2c33cd47e19092/check-versions.sh#L24-L27 (just running `exiftool` or `exiftool -v` or `exiftool --version`  logs waaay to much info. it logs the entire documentation)

Comment (jdelbick) [reply check-versions.sh:22]: ah I figured it out:
```
exiftool -ver
```

**Review by tmathern — APPROVED** [review]

File: check-versions.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# log exiftool version
+echo "....................................................................................................."
+echo "Exiftool Version:"
+exiftool -ver
+echo "....................................................................................................."
+
+met=$(exiftool README.md -json | grep "FileSize")
+if [[ -z "$met" ]]; then
+    echo "WARNING: Exiftool not installed."
+fi
```
Comment (pheenomenon) [inline check-versions.sh:34]: Can remove this? `met=$(exiftool README.md -json | grep "FileSize")`

**Review by pheenomenon — APPROVED** [review]

File: check-versions.sh (hunk 3 above)
Comment (jdelbick) [reply check-versions.sh]: Yep, leftover from before I discovered how to log exiftool version. Nice catch!

**Pull request merged**